Dockerfile-controllers.test

```diff
@@ -23,9 +23,11 @@ USER default
 RUN pip3 install --ignore-installed --default-timeout=100 --upgrade pip==21.2.4
 WORKDIR /tmp
 COPY controllers/servers/csi/requirements.txt .
+COPY controllers/servers/host_definer/requirements.txt ./host_definer_requirements.txt
 # avoid default- boringssl lib, since it does not support z systems
 ENV GRPC_PYTHON_BUILD_SYSTEM_OPENSSL=True
 RUN pip3 install -r ./requirements.txt
+RUN pip3 install -r ./host_definer_requirements.txt
 
 USER root
 COPY controllers/scripts/csi_general .
```

Comment:
I think
`controllers/servers/csi/requirements-tests.txt` should move to:
`controllers/servers/requirements-tests.txt`
or better yet:
`controllers/tests/requirements.txt`

Reply:
done